packages-native/popup-menu/src/PopupMenu.tsx

```diff
@@ -29,50 +28,52 @@ export function PopupMenu(props: PopupMenuProps<PopupMenuStyle>): ReactElement {
         [menuRef.current]
     );
 
-    const Touchable: ComponentType<TouchableNativeFeedbackProps | TouchableHighlightProps> =
-        Platform.OS === "android" ? TouchableNativeFeedback : TouchableHighlight;
+    const Touchable: ComponentType<TouchableNativeFeedbackProps | TouchableOpacity> =
+        Platform.OS === "android" ? TouchableNativeFeedback : TouchableOpacity;
 
     let menuOptions: ReactElement[];
     if (props.renderMode === "basic") {
-        menuOptions = props.basicItems.map((item, index) =>
-            item.itemType === "divider" ? (
-                <MenuDivider key={index} color={styles.basicItem?.dividerColor} />
+        menuOptions = props.basicItems.map((item, index) => {
+            const itemStyle = styles.basic?.itemStyle && styles.basic?.itemStyle[item.styleClass];
```

Comment:
This expression returns boolean

Reply:
Use ternary instead

Reply:
If i type on console
let x = "",
let y = {z: "yolo1"}
x = y && y.z
x becomes "yolo1"

Reply:
ok then